src/CONST.js

```diff
@@ -790,6 +790,9 @@ const CONST = {
         INVITE: 'invite',
         LEAVE_ROOM: 'leaveRoom',
     },
+
+    MAX_64BIT_TOP_HALF: 9223372036,
```

Comment:
Isn't 9223372036 the bottom half? 😕 Or is this name confusing (me)?

Reply:
Heh, we must think of it differently, I see it as the bottom half (the bottom of the number) 😄 Left and right sounds good to me

Reply:
Yea, we can change it to left and right 👍